spec/data/node_spec.rb

```diff
@@ -331,6 +331,21 @@ def canonicalize(hw_info)
       node.installed.should == 'test'
     end
 
+    it "should update tags if the node is marked installed" do
```

Comment:
I'd actually move this test out of "binding on checkin" since this does not test binding at all.